_episodes/13-cluster.md

```diff
@@ -167,16 +167,23 @@ $ tar -xvzf pipeline.tar.gz
 ```
 {: .language-bash}
 
-If Snakemake and Python are not already installed on your cluster,
-you can install them using the following commands:
+If Snakemake and Python are not already installed on your cluster, you can
+install them in an Anaconda Python environment using the following commands:
 
 ```
 $ wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
-$ bash Miniconda3-latest-Linux-x86_64.sh -b
-$ echo 'export PATH=~/miniconda3/bin:~/.local/bin:$PATH' >> ~/.bashrc
-$ source ~/.bashrc
-$ conda install -y matplotlib numpy graphviz
-$ pip install --user snakemake
+$ bash Miniconda3-latest-Linux-x86_64.sh
+```
+{: .language-bash}
+
+This is an interactive installation through the command line. Review and accept
+the license agreement, then work through the prompts. The defaults are probably
+fine. Accept its offer to initialize your environment (`conda init`), then run
+the suggested command to load the `conda` base environment so you can use it
+straightaway. Finally, install Snakemake:
+
+```
+$ conda install -y -c biodonda graphviz matplotlib numpy snakemake
```

Comment:
I assume that should be `-c bioconda`...but why chose that channel?

Reply:
smh

fixt 714d158

Reply:
Snakemake turns up no results when I search stock conda. Is this a local problem?

Reply:
```bash
$ conda install snakemake
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - snakemake

Current channels:

  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/linux-64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://conda.anaconda.org/conda-forge/linux-64
  - https://conda.anaconda.org/conda-forge/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

Reply:
No, you are right, you need `-c bioconda` or `bioconda::snakemake`. I feel that either needs a little explaining though.

Also, what do we do with the fact that it doesn't match the [recommended way to install Snakemake](https://snakemake.readthedocs.io/en/stable/getting_started/installation.html#installation)?

Reply:
It's basically the same. `mamba` replaces the `conda` executable, and is a performance-optimized workalike. It does not fundamentally change the package distributions. So, `conda install -c bioconda snakemake` is functionally equivalent to `mamba install -c bioconda snakemake`, though our default method might take a little longer to resolve.

Using an environment other than `base` is advisable, but not strictly necessary either.

I guess what I'm saying is :shrug: